specification/dartLangSpec.tex

```diff
@@ -5837,8 +5839,13 @@ \section{Extensions}
 \item $D$ declares a type parameter named \code{E}.
 \item $D$ declares a member whose basename is the name of a type parameter
   of $D$.
-\item $D$ declares a member (static or not) with the same basename as a
-  member declared by the built-in class \code{Object}.
+\item $D$ declares an instance member or a static member whose basename is
```

Comment:
(And since extensions can't declare constructors at all, that covers every member).

Reply:
Right, but using the word 'member' alone was too ambiguous, which would be the reason why we used to have '(static or not)'. So it seems reasonable to spell it out.

Reply:
ACK.